Chinese/Yaoming.md

```diff
@@ -0,0 +1 @@
+转眼姚明都已经退役好几年了，科比也要退役了，老一代基本剩不下什么人了，伊戈达拉都要退役了，真的是老了。
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.